### PR TITLE
Support additional_emails field on the Customer type

### PR DESCRIPTION
**Pull request by sjmiller609**
Support additional_emails on the Customer type

Orb reference docs

- Create https://docs.withorb.com/reference/create-customer
- Update https://docs.withorb.com/reference/update-customer

### CODE REVIEW

File: src/client/customers.rs [hunk 1]
```diff
@@ -152,6 +158,8 @@ pub struct Customer {
     pub name: String,
     /// A valid email for the customer, to be used for notifications.
     pub email: String,
+    /// Additional email addresses for this customer.
+    pub additional_emails: Vec<String>,
```
Comment (sjmiller609) [inline src/client/customers.rs:162]: In orb doc it says required field in response body - need to confirm always returned in response payload even if empty list

**Comment by sjmiller609** [comment]
@benesch Thank you for the library and accepting PRs, let me know anything else! I didn't actually run the tests because of the emails and Orb key, but if they pass it should be good. Please let me know anything else!

File: src/client/customers.rs (hunk 1 above)
Comment (sjmiller609) [reply src/client/customers.rs:162]: covered by test

**Comment by benesch** [comment]
Thanks for the PR! I triggered a CI run. @arusahni could you handle the review here?

**Comment by sjmiller609** [comment]
The test failure is related to GHA not sharing tokens across forks. I can switch to a branch if you grant me writer to the repo, that would dodge the issue, or maybe there's another way?

**Comment by arusahni** [comment]
Ah, yeah, that's annoying. I can pull this down and run the Linux tests locally, and we can do code review on this PR. Once it looks good to merge I'll open a branched PR with your commits (preserving your authorship) and we can get the final CI run and merge this in. Does that sound okay to you?

**Comment by sjmiller609** [comment]
> Ah, yeah, that's annoying. I can pull this down and run the Linux tests locally, and we can do code review on this PR. Once it looks good to merge I'll open a branched PR with your commits (preserving your authorship) and we can get the final CI run and merge this in. Does that sound okay to you?

Sounds good! Just let me know if you change you mind, I can follow another process to make it easier for you if I have another change request later.

**Comment by arusahni** [comment]
No worries at all! We very much appreciate your contributions :-)

I'll try to get to this within the next day or so.

**Review by arusahni — COMMENTED** [review]
This looks good! I can verify that, with the typo fix I suggested, the tests pass. Let's get a few updates made and then we can close it for me to resubmit via a branch:

1. Revert the `Cargo.toml` version bump.
2. Apply the email address fix suggestion.
3. Add an entry to `CHANGELOG.md` for this. Since we're using Cargo release, [we use this convention for recording updates](https://github.com/crate-ci/cargo-release/blob/master/docs/faq.md#maintaining-changelog). Spefically, place the entry under the "Unreleased" header. It will automatically be associated with the new version when we cut the release.
4. Let's squash the commits into a single one with a semantic commit message.

**Comment by sjmiller609** [comment]
@arusahni ok done

**Pull request merged**